dev/com.ibm.ws.logging_fat/publish/servers/com.ibm.ws.logging.stackjoiner/server.xml

```diff
@@ -0,0 +1,22 @@
+<!--
+    Copyright (c) 2020 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server description="Server for testing Liberty logging in a server">
+
+    <include location="../fatTestPorts.xml"/>
+
+    <featureManager>
+	   <feature>jsp-2.3</feature>
+    </featureManager>
+    
+    <javaPermission className="java.util.PropertyPermission"  name="*" actions="read" />
```

Comment:
Why is this needed?  It shouldn't be needed from what I can see for the application that is used.

Reply:
fixed in [96faaf5](https://github.com/OpenLiberty/open-liberty/pull/16276/commits/96faaf592d1ddeb0455f5be1d4ad792995503d05)